projects/eudsl-python-extras/pyproject.toml

```diff
@@ -12,3 +12,8 @@ log_cli = false
 log_cli_level = "DEBUG"
 log_cli_format = "[%(filename)s:%(funcName)s:%(lineno)d] %(message)s"
 log_cli_date_format = "%Y-%m-%d %H:%M:%S"
+
+[tool.ruff.lint]
+ignore = [
+    "F405", # value from star import
+]
```

Comment:
my strong (strong) preference is blacklist versus whitelist - we used ruff internally and i hate it with the fire of a thousand suns. i know you haven't turned it on yet but just saying...